### ISSUE TITLE
Google maps does not always works

### ISSUE THREAD

**Issue by dudarev** [dudarev/mapjumper#38]
When Google maps is loaded from scratch icon is not shown:

Errorr is console:

```
Failed to load resource: net::ERR_CACHE_MISS
```

Reload helps.

**Comment by ignaciolep** [comment]
The extension doesn't work with Google Maps. When will be 0.7 released?

**Comment by dudarev** [comment]
If the extension does not work for you even for after reloading the page with Google Maps, it may be because maps URL is not included in `content_script.matches`: https://github.com/dudarev/mapjumper/blob/master/src/manifest.json#L13
They will need to be updated to more general.

Regarding the issue that can be fixed by reloading: it's not clear for me how it can be fixed yet.

I'll try to find time to work on 0.7 and ship it by the end of 2017.

**Comment by ignaciolep** [comment]
You are right, Google redirects me to www.google.com.ar/maps/ (For example: https://www.google.com.ar/maps/@-34.6042668,-58.3804513,15z) and that TLD is missing in content_script.matches.

The issue #29 suggest adding a lot of missing Google TLDs. Can't you use a wildcard for that? For example: `"*://www.google.*/maps/*"`

**Comment by dudarev** [comment]
Yes, that's what I've just thought too and created an issue #52. I'll test
if it works.
